Recursive/Palindrome.js

```diff
@@ -1,12 +1,14 @@
+/**
+ * @function Palindrome
+ * @description Check whether the given string is Palindrome or not.
+ * @param {String} str - The input string
+ * @return {Boolean}.
+ * @see [Palindrome](https://en.wikipedia.org/wiki/Palindrome)
+ */
 
-// Check whether the given string is Palindrome or not
-export const Palindrome = (str) => {
-  if (typeof str !== 'string') {
-    str = str.toString()
-  }
-
-  if (str === null || str === undefined) {
-    return false
+const palindrome = (str) => {
+  if (typeof str !== 'string' || str === null || str === undefined) {
```

Comment:
The comparisons to `null` and `undefined` are redundant as if `str` is `null` or `undefined`, it's `typeof` would not be `"string"`.

Reply:
Fix